### DISCUSSION TITLE
Upload 可以结合webuploader 实现分片上传吗

### DISCUSSION THREAD

**Discussion by hj12hj** [ant-design/ant-design#33985] [General]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
大文件上传慢的问题

### What does the proposed API look like?
可以自定义上传

**Comment by MadCcc** [comment]
Did you mean `customRequest` ?
